security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml

```diff
@@ -1098,6 +1098,21 @@
         <type>textbox</type>
         <help>Requires the whole key file in a format that is compatible with TransIP.</help>
     </field>
+    <field>
+        <label>udr</label>
```

Comment:
This is a bit cryptic for a label. Would you mind to relabel it to "united-domains Reselling"? :)

Reply:
Done :)